forest/src/cli/config.rs

```diff
@@ -53,6 +53,15 @@ impl LogValue {
     }
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Eq, Default)]
+pub struct SnapshotFetchConfig {
+    pub mainnet_snapshot_url: Option<String>,
+    pub calibnet_snapshot_spaces_url: Option<String>,
+    pub calibnet_bucket_name: Option<String>,
+    pub calibnet_region: Option<String>,
+    pub calibnet_path: Option<String>,
+}
+
```

Comment:
To give those a bit of structure (hopefully not over-engineering), I'd put networks in their respective structs; this way they'd be less coupled, and we wouldn't have to prefix every parameter. What do you think?

Additionally, it'd be nice for those parameters to be self-contained; it's a bit odd to have the default value for e.g. calibnet bucket name in a completely separate file than the struct. What do you think about adding the constants from `snapshot_fetch.rs` as parts of `Default` trait implementation for this struct? They wouldn't have to be `Option`s anymore.

The icing on the cake would be having both `mainnet_snapshot_url` and `calibnet_snapshot_spaces_url` typed with `Url` and not `String` And if you add documentation specifying what each of those parameters does I'd be extra grateful. Please ping me if you need help formulating those.

Reply:
Fixed as outlined.